src/interaction/orbitalnavigator.cpp

```diff
@@ -263,15 +319,58 @@ void OrbitalNavigator::updateCameraStateFromMouseStates(Camera& camera, double d
         // Update the camera state
         camera.setPositionVec3(camPos);
         camera.setRotation(camRot.globalRotation * camRot.localRotation);
+
+        if (_useAdaptiveStereoscopicDepth) {
+            double targetCameraToSurfaceDistance = glm::length(
+                cameraToSurfaceVector(camPos, centerPos, posHandle)
+            );
+            if (_directlySetStereoDistance) {
+                _currentCameraToSurfaceDistance = targetCameraToSurfaceDistance;
+                _directlySetStereoDistance = false;
+            } else {
+                _currentCameraToSurfaceDistance = interpolateCameraToSurfaceDistance(
+                    deltaTime,
+                    _currentCameraToSurfaceDistance,
+                    targetCameraToSurfaceDistance);
+            }
+
+            camera.setScaling(
+                _stereoscopicDepthOfFocusSurface /
+                static_cast<float>(_currentCameraToSurfaceDistance)
+            );
+        } else {
+            camera.setScaling(glm::pow(10.f, _staticViewScaleExponent));
+        }
     }
 }
 
+glm::dvec3 OrbitalNavigator::cameraToSurfaceVector(
+    const glm::dvec3& camPos,
```

Comment:
Parameter sorting per line

Reply:
?

Reply:
For function definitions in the cpp file that don't fit on one line, we would have:

```cpp
ReturnType foobar(const T& t, const T& s, const T& t,
                                  const std::vector<T>& foo)
{
  // ..
}
```

Reply:
Fixed.